Modules/Core/Common/wrapping/itkVectorContainer.wrap

```diff
@@ -4,8 +4,11 @@ itk_wrap_class("itk::VectorContainer" POINTER)
   # double is always needed by KernelTransform
   UNIQUE(scalar_types "${WRAP_ITK_SCALAR};D")
   foreach(t ${scalar_types})
-    itk_wrap_template("${ITKM_IT}${ITKM_${t}}" "${ITKT_IT},${ITKT_${t}}")
-    itk_wrap_template("${ITKM_UC}${ITKM_${t}}" "${ITKT_UC},${ITKT_${t}}")
+    # These are wrapped below with "vectypes"
+    if(NOT ${t} STREQUAL "UC")
```

Comment:
why is "UC" not wrapped? I don't see anywhere else where it is wrapped in this file.

Reply:
UC is wrapped here: https://github.com/InsightSoftwareConsortium/ITK/pull/442/files#diff-f6ed0a76b0d14eeb12481dde620b2ef6L30

If it was not wrapped a build error or warning would result.